homeassistant/components/growatt_server/sensor.py

```diff
@@ -817,6 +974,10 @@ def update(self):
             elif self.growatt_type == "inverter":
                 inverter_info = self.api.inverter_detail(self.device_id)
                 self.data = inverter_info
+            elif self.growatt_type == "tlx":
+                tlx_info = self.api.tlx_detail(self.device_id)
+                self.data = tlx_info["data"]
+                _LOGGER.debug(tlx_info["data"])
```

Comment:
Can we remove this one please? We don't have it for any of the others (appreciate it was probably handy for you to have this when implementing the tlx logic though)